### ISSUE TITLE
Typos and grammar mistakes in German translation

### ISSUE THREAD

**Issue by rillig** [github/docs#26541]
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/de/site-policy/github-terms/github-terms-of-service

### What part(s) of the article would you like to see updated?

Mismatch of Sie/du (occurs multiple times):
> Wir freuen uns, dass Sie hier sind. Bitte lies diese

Duplicate word:
> Bitte lies diesen Abschnitt bitte

Wrong article, plus the space is underlined:
> findest du in unserem[ Exportkontrollrichtlinie](https://docs.github.com/de/site-policy/other-site-policies/github-and-trade-controls).

In summary, please let a human proofread the complete article. I intentionally did not list all the typos and grammar mistakes I found. Don't trust a machine translation. As you say yourself:
> Da es sich um einen so wichtigen Vertrag zwischen uns und unseren Nutzern handelt, […]

If this contract is really important to you, you should ensure that it is free of spelling and grammar mistakes. Why should I trust the wording if the spelling is incorrect?

**Comment by cmwilson21** [comment]
👋 Hey @rillig - Thank you so much for opening this followup issue! I'll get it flagged and submitted for review internally. So I'll go ahead and close this out. But it's really helpful to be able to link to this issue ✨

**Comment by rillig** [comment]
Any updates on this topic? If the document is indeed as important as claimed, it should get more attention and at least appear trushworthy.
